tests/berlin/eip2929_gas_cost_increases/test_call_gas_cost_change.py

```diff
@@ -0,0 +1,349 @@
+            _address = hex(address)
+            precompile_exists = True
+            yield pytest.param(
+                _address,
+                precompile_exists,
+                id=f"address_{_address}_precompile_{precompile_exists}",
+            )
+        elif num_unsupported > 0:
+            # Check unsupported precompiles up to NUM_UNSUPPORTED_PRECOMPILES
+            _address = hex(address)
+            precompile_exists = False
+            yield pytest.param(
+                _address,
+                precompile_exists,
+                id=f"address_{_address}_precompile_{precompile_exists}",
+            )
+            num_unsupported -= 1
+
+
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize_by_fork("address,precompile_exists", call_addresses_and_expected_output)
+def test_call_params_value_0(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    address: str,
+    precompile_exists: int,
+):
+    """Tests equal gas consumption of CALL operation for the given addresses and parameters."""
+    env = Environment()
+
+    account = pre.deploy_contract(
+        initial_setup_bytecode(pre.fund_eoa(amount=0)) + contract_bytecode(address=address),
+        storage={0: 0xDEADBEEF},
+    )
+
+    tx = Transaction(
+        to=account,
+        sender=pre.fund_eoa(),
+        gas_limit=1_000_000,
+        protected=True,
+    )
+
+    expected_gas_cost_change = hex(0x0)
+    if not precompile_exists:
+        expected_gas_cost_change = hex(0x09C4)
+    elif fork == Cancun and address in ["0x9", "0xa"]:
+        expected_gas_cost_change = hex(0xDEADBEEF)
+    elif fork == Prague and address in [
+        "0x9",
+        "0xa",
+        "0xb",
+        "0xc",
+        "0xd",
+        "0xe",
+        "0xf",
+        "0x10",
+        "0x11",
+    ]:
+        expected_gas_cost_change = hex(0xDEADBEEF)
+
+    post = {account: Account(storage={0: expected_gas_cost_change})}
+
+    state_test(env=env, pre=pre, post=post, tx=tx)
+
+
+@pytest.mark.valid_from("Cancun")
+@pytest.mark.parametrize_by_fork("address,precompile_exists", call_addresses_and_expected_output)
+def test_call_params_value_1(
+    state_test: StateTestFiller,
+    pre: Alloc,
+    fork: Fork,
+    address: str,
+    precompile_exists: int,
+):
+    """Tests equal gas consumption of CALL operation for the given addresses and parameters."""
+    env = Environment()
+
+    account = pre.deploy_contract(
+        initial_setup_bytecode(pre.fund_eoa(amount=0))
+        + contract_bytecode(address=address, value=1),
+        storage={0: 0xDEADBEEF},
+    )
+
+    tx = Transaction(
+        to=account,
+        sender=pre.fund_eoa(),
+        gas_limit=1_000_000,
+        protected=True,
+    )
+
+    expected_gas_cost_change = hex(0x0)
+    if not precompile_exists:
+        expected_gas_cost_change = hex(0x6B6C)
+    elif fork >= Cancun and address in ["0x1", "0x2", "0x3", "0x4", "0x5", "0x6", "0x7", "0x8"]:
```

Comment:
The fork class iteslf has a list of precompiles defined in it. Use that function

Reply:
The logic in each of these tests is set for only certain precompiles, for example the precompiles at `0x9` and `0xa` for Cancun should expect to have no change in the cost of gas where the others will expect to have a cost difference of `0x61A8`.